model/distroqueue/distro_queue.go

```diff
@@ -0,0 +1,21 @@
+package distroqueue
```

Comment:
is this a new package? 

maybe it makes sense to put this in top-level model directory with the other task queue structure

Reply:
Moved to the *model* package.